packages/foreman/foreman/foreman.spec

```diff
@@ -1274,6 +1271,9 @@ exit 0
 %systemd_postun_with_restart %{name}.service
 
 %changelog
+* Mon Jul 16 2018 Eric D. Helms <ericdhelms@gmail.com> - 1.18.0-2
+- rebuilt
```

Comment:
I think we could be a bit more descriptive here :)

Reply:
I can update this.